src/AggregateFunctions/AggregateFunctionGroupBitmapData.h

```diff
@@ -151,10 +151,12 @@ class RoaringBitmapWithSmallSet : private boost::noncopyable
         }
         else if (BitmapKind::Bitmap == kind)
         {
-            auto size = roaring_bitmap->getSizeInBytes();
+            std::shared_ptr<RoaringBitmap> bitmap = std::make_shared<RoaringBitmap>(*roaring_bitmap);
```

Comment:
Do we need to make it shared_ptr?

Reply:
I don't think, it's necessary to be shared.

Reply:
If we can make it unique_ptr or even better - allocated on stack it should save us a few allocations.

Reply:
Not sure about stack.
https://github.com/RoaringBitmap/CRoaring/issues/112#issuecomment-385924079

Plus, bitmaps can be huge.

But, i have no objections against unique_ptr